src/bank-holidays/bank-holidays.constant.js

```diff
@@ -0,0 +1,230 @@
+export const BANK_HOLIDAYS = {
```

Comment:
Don't we have an API for this ? Isn't there a library for this ?

Reply:
Wouldn't this work ? https://github.com/commenthol/date-holidays

Reply:
I created this service to to have the bank holidays which will be used by two controllers into the manager telecom (TelecomTelephonyAliasPortabilityOrderCtrl and TelephonySchedulerBankHolidaysCtrl).

Reply:
I understand what you did :)

Don't you think this shouldn't be done by us ? Does anyone in the API team work on a solution to be able to retrieve the bank holidays from them ?